newlib/libc/tinystdio/fputs.c

```diff
@@ -36,17 +36,22 @@ fputs(const char *str, FILE *stream)
 {
```

Comment:
Why does this file not follow the same pattern of wrapping a static inline with the locking code?

Reply:
refactored